Added update company profile details (#246)
companieshouse/test-data-generator: README.md

  - A usage example for creating a company with advanced search: `{ "advanced_search": true }`
  - A usage example for creating a company with disqualified officers: `{ "disqualified_officers": [ { "disqualification_type": "court-order", "is_corporate_officer": false }]}`

> - PUT: Sending a PUT request to update Company Profile for a specific company `{Base URL}/test-data/internal` will update an Company Profile entry. The request body must include mandatory `company_number`and optional `etag` parameters.

Reviewer: ```suggestion
- PUT: Sending a PUT request to update Company Profile for a specific company `{Base URL}/test-data/internal` will update a Company Profile entry. The request body must include `company_number` and can include optional `etag` parameter.
```

Author: updated

> - PUT: Sending a PUT request to update Company Profile for a specific company `{Base URL}/test-data/internal` will update an Company Profile entry. The request body must include mandatory `company_number`and optional `etag` parameters.

Reviewer: ```suggestion
- PUT: Sending a PUT request to `{Base URL}/test-data/internal` will update a Company Profile entry. The request body must include mandatory `company_number`and optional `etag` parameters.
```
